core/src/main/scala/org/apache/spark/ui/JettyUtils.scala

```diff
@@ -186,6 +188,50 @@ private[spark] object JettyUtils extends Logging {
     contextHandler
   }
 
+  /** Create a handler for proxying request to Workers and Application Drivers */
+  def createProxyHandler(
+      prefix: String,
+      target: String): ServletContextHandler = {
+    val servlet = new ProxyServlet {
+      override def rewriteTarget(request: HttpServletRequest): String = {
+        val rewrittenURI = createProxyURI(
+                              prefix,
```

Comment:
nit: style. If arguments don't fit in one line, they should be indented two levels (= 4 spaces).